### PR TITLE
Update Travis config to use latest macOS; lock more gems for system-ruby compatibility

### PR DESCRIPTION
**Pull request by aramprice**

### CODE REVIEW

**Comment by aramprice** [comment]
@wendorf or @cunnie would you mind looking this over.

**Comment by aramprice** [comment]
... sadly I broke master with my [README update](https://github.com/pivotal-sprout/sprout-wrap/commit/b9a51d127836887fc0ad2dd094fbc24ed4070e9e) because of system-ruby-incompatible gems :-/

This should fix the build, in addition to making CI run on the latest macOS.

File: .travis.yml [hunk 1]
```diff
@@ -1,24 +1,15 @@
...
-  - brew cask list
-  - brew cask zap $(brew cask list)
-  - brew list
-  - brew uninstall --force $(brew list)
-  - ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall)"
-
-  # We are testing installation of Rubies, so should uninstall everything
-  - rvm implode --force
+  - for ruby in $(rvm list strings); do rvm uninstall "${ruby}"; done
+  - travis_wait rvm implode --force # We are testing installation of Rubies, so should uninstall everything
```
Comment (aramprice) [inline .travis.yml]: The addition of `travis_wait` here should fix the current build timeout/failure on master. See https://travis-ci.org/pivotal-sprout/sprout-wrap/builds/222205316#L431

**Comment by wendorf** [comment]
Looks reasonable to me, though I didn't run it. Pretty disappointing that we have to lock so many gems, though.

**Comment by cunnie** [comment]
Looks reasonable to me, too, but I'm very removed from the day-to-day sprout-wrap, so take my suggestion with a grain of salt.

**Comment by aramprice** [comment]
@cunnie would you care to hit the merge button or shall I?

**Comment by cunnie** [comment]
It's 4am in Bali, I'm wide awake and my friend is fast asleep, so why not take this opportunity to merge a PR?

**Comment by aramprice** [comment]
BALI?! that's awesome... but sorry to disrupt your vacation :-/

Have a great rest of the trip!

**Comment by aramprice** [comment]
This is currently failing because the OS upgrades triggered by the sprout run are timing out (10+ minutes)

:-/

**Comment by cunnie** [comment]
Travis failing due to overly-long OS upgrades? Sigh, well that's not a real fail in my book.

**Comment by trinitronx** [comment]
@cunnie:  Maybe [this helps](https://docs.travis-ci.com/user/common-build-problems/#My-builds-are-timing-out)?

**Pull request merged**